### ISSUE TITLE
[Accessibility][Tracking] Interactive Feedback Announcements

### ISSUE THREAD

**Issue by rushatgabhane** [Expensify/App#84310]
## Overview

This issue tracks missing screen reader announcements after user-initiated interactions. Users should receive audible feedback when interacting with UI elements like emoji categories and context menus.

## Context

Part of the accessibility improvement effort tracked in https://github.com/Expensify/Expensify/issues/575078

## Issues to Fix

- https://github.com/Expensify/App/issues/77490 - No announcement after selecting emoji category
- https://github.com/Expensify/App/issues/76948 - No information of context menu

## Instructions

**Suggested Fix:** Add appropriate `accessibilityLabel` / `aria-label` and announce state changes after user interactions (e.g., selected category name, context menu availability).

> **Note:** Discuss on Slack if you are unsure whether a sub-issue should be fixed or if you are unsure about the right fix / expected result. Ideally, create a single PR that will fix all the issues listed to minimize review time (more PRs only if necessary).

<details><summary>Issue Owner</summary>Current Issue Owner: @truph01</details>

**Comment by rushatgabhane** [comment]
@truph01 btw, for one of these issue this PR is there - https://github.com/Expensify/App/pull/82932

You can C+ approve it if it looks good. Otherwise continue work on this

**Comment by truph01** [comment]
We can close this one since all child issues are done.

**Comment by truph01** [comment]
@rushatgabhane Could you close this one?
